docs/references/metrics.mdx

```diff
@@ -95,8 +95,7 @@ models:
               sql: 'IF(${revenue} IS NULL, 10, ${revenue})'
               groups: ['product_details', 'item_details'] # this would add the metric to a nested group: `product details` --> `item details`
               hidden: false
-              round: 0
-              format: 'gbp'
+              format: '"£"#,##0.00' # GBP rounded to two decimal points
```

Comment:
For currency use `[$<currency symbol>]` instead of text prefix `"£"`
```suggestion
              format: '[$£]#,##0.00' # GBP rounded to two decimal points
```